src/main/java/io/icker/factions/FactionsMod.java

```diff
@@ -26,7 +26,7 @@ public class FactionsMod implements ModInitializer {
 
     @Override
     public void onInitialize() {
-        LOGGER.info("Initialized Factions Mod for Minecraft v1.19");
+        LOGGER.info("Initialized Factions Mod for Minecraft v1.19.2");
```

Comment:
Since this message is very rarely seen there's no real need to specify the minor version, its only another thing we have to change if an update is released